system-test/testnet-automation.sh

```diff
@@ -223,6 +223,9 @@ if [[ -z $NUMBER_OF_VALIDATOR_NODES ]]; then
   exit 1
 fi
 
+dbName="$TESTNET_TAG"
```

Comment:
So won't clobber the testnets, but you will have overlapping writes to the same db in this case if two hosts are running parallel networks with the same `TESTNET_TAG`

Reply:
You'll need to update `init-metrics.sh` here https://github.com/solana-labs/solana/blob/13ff6b2f218f185590f058fb0f56495e683f7e78/system-test/testnet-automation.sh#L106 to support the dynamic creation of new db's for this to work.

Reply:
Ah... yeah... are we worried about having a mess of DBs?

Would be nice if we could have just the one, then generate a UUID for each test run that's tagged to metrics, but I suspect that's a lot of plumbing elsewhere

Reply:
Could set custom retention policies for dynamically-created DBs so they don't stick around for too long.  The slackbot already creates a custom URL for each test that's run so it displays the db in question only for the time during which the test was running. If something simple like that works for you, that's fine.  The bigger problem will come if/when you are having nodes from unrelated testnets writing to the same db at the same time.

Reply:
Thanks!  I'll look into that this evening and report back